front-end/src/types/index.ts

```diff
@@ -1,12 +1,33 @@
 import { Key } from "react";
 
 interface TodoType {
-    createdAt: string,
-    createdBy: string,
-    description: string,
-    needsReminder: boolean,
-    title: string,
-    id: Key
+    createdAt: string;
+    createdBy: string;
+    description: string;
+    needsReminder: boolean;
+    title: string;
+    id: Key;
+}
+
+export interface BranchType {
```

Comment:
As the codebase scales, it would have large number of types, add types for every feature in different file and then export it. types for statistics feature can be added in a file named statistics.ts  and then you can export it from index .ts like this
export {BranchType } from './statistics.ts'